bcgov/mds
# Feature/mds 3995 feature flagging in minespace

# Main

- Adds feature flagging to MineSpace through the AuthorizationWrapper Component just as it is in CORE.

# Other

- Added the flagging to the Record Incident button in MineSpace to shoe in test.

# How to test

- See that the button is available in test and then push to prod and confirm the button is not there.
